manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -361,6 +361,25 @@ properties:
     links:
       passwd: (( concat "https://login." properties.system_domain "/forgot_password" ))
       signup: (( concat "https://login." properties.system_domain "/create_account" ))
+    oauth:
+      providers:
+        google:
```

Comment:
I believe, we'd need to edit all entries in the database and rename their provider name from `google` to `google_gds`.

Agree, it should be a more related alias name.